pkg/net/http/loggingMiddleware.go

```diff
@@ -28,7 +34,25 @@ func (w *statusWriter) Write(b []byte) (int, error) {
 	if w.status == 0 {
 		w.status = 200
 	}
+	if w.status >= 400 && len(b) > 0 && w.err == nil {
+		switch w.ResponseWriter.Header().Get(uri.ContentTypeHeaderKey) {
+		case uri.ApplicationProtoJsonContentType:
+		case message.AppJSON.String():
+			var s rpcStatus.Status
+			if err := protojson.Unmarshal(b, &s); err == nil {
+				w.err = status.ErrorProto(&s)
+			}
+		}
+		if w.err == nil {
+			errLen := 1024
```

Comment:
Is the header limited to this size or why the truncation of the error?

Reply:
I want to just limit size of error message - it can be big